CMakePresets.json

```diff
@@ -34,7 +34,54 @@
             "name": "ninja-vcpkg",
             "configurePreset": "ninja-multi-vcpkg",
             "displayName": "Build",
-            "description": "Build with Ninja/vcpkg"
+            "description": "Build with Ninja/vcpkg",
+            "hidden": true
+        },
+        {
+            "name": "ninja-vcpkg-check-format",
+            "configurePreset": "ninja-multi-vcpkg",
+            "displayName": "Build",
```

Comment:
could the `displayName` be renamed to `check-format`? In principle to me it looks we are not building the .cpp files.